llvm/lib/Target/X86/X86CompressEVEX.cpp

```diff
@@ -220,6 +220,27 @@ static bool performCustomAdjustments(MachineInstr &MI, unsigned NewOpc) {
   return true;
 }
 
+static bool isRedundantNewDataDest(MachineInstr &MI, const X86Subtarget &ST) {
+  // add rbx, rbx, rax / add rbx, rax, rbx -> add rbx, rax
```

Comment:
It would be better if you use a real node, e.g., ADDrr_ND instead of a add.

Reply:
Done